text/0069-api-v2.md

```diff
@@ -93,6 +93,7 @@ The keyspaces are stored in etcd and has the no limitation on the id number.
 3. Flashbacking keyspace:
   3.1. Flashbacking keyspace only affects the metadata, turning the keyspace name and id mapping to viable by clients.
   3.2. If there are multiple deleted keyspaces with the same name, only the last deleted keyspace is flashbacked. Users can flashback all these deleted keyspaces by calling the flashback API multiple times with different new keyspace names.
+4. Reserving keyspace name: "default".
```

Comment:
could you elaborate on what's 'default' keyspace is in the RFC?

Reply:
After consideration, I've removed it.